### ISSUE TITLE
Deterministic counter of categorical variables

### ISSUE THREAD

**Issue by nemuo** [bayespy/bayespy#93]
Hi, this bayespy is very interesting. Let me ask another question.

Let's say two people (c and c2) choose A or B at random. I would like to know how many people choose A.

I tried to make a node of the counter as follows.

```
c = Categorical([0.2, 0.8]);
c2 = MultiMixture((c,), Categorical, [[0.9, 0.1],[0.3,0.7]]);
m = MultiMixture((c,), Multinomial, 1,[[1.0,0.0],[0.0,1.0]]);
print(m.random());
m2 = MultiMixture((c2,), Multinomial, 1,[[1.0,0.0],[0.0,1.0]]);
cnt = Add(m,m2);
```

The result is as follows. Is it possible to make a deterministic counter of categorical nodes?

```
[-2147483647           0]
Traceback (most recent call last):
  File "./bayespytest.py", line 317, in <module>
    cnt = Add(m,m2);
  File "C:\home\bin\python3_32\lib\site-packages\bayespy-0.5.6-py3.5.egg\bayespy\inference\vmp\nodes\add.py", line 64, in __init__
    node = self._ensure_moments(node, GaussianMoments, ndim=None)
  File "C:\home\bin\python3_32\lib\site-packages\bayespy-0.5.6-py3.5.egg\bayespy\inference\vmp\nodes\node.py", line 364, in _ensure_moments
    converter = node._moments.get_converter(moments_class)
  File "C:\home\bin\python3_32\lib\site-packages\bayespy-0.5.6-py3.5.egg\bayespy\inference\vmp\nodes\node.py", line 180, in get_converter
    moments_to.__name__))
bayespy.inference.vmp.nodes.node.NoConverterError: No conversion defined from MultinomialMoments to GaussianMoments
```

**Comment by jluttine** [comment]
Adding multinomial/categorical variables is not yet supported, although it would be quite a simple and straightforward task. I'll add it to TODO list. `Add` node currently only supports adding Gaussian nodes. That's what the cryptic error message is about.

If you weren't interested in the individuals' decisions and all individuals had the same decision probabilities, then you could just directly use one multinomial with as many trials as individuals. Apparently, that's not the case here now.

Just a minor comment: it's not necessary to use `MultiMixture` as you only have one mixing variable in the mixture distributions (either `c` or `c2`). For instance:
```
c2 = Mixture(c, Categorical, [[0.9, 0.1],[0.3,0.7]])
```

So to sum up: I'll make this issue a feature request for `Add` support for multinomial/categorical nodes.

**Comment by nemuo** [comment]
Thank you.
I look forward to it.
